app-editors/gvim/gvim-9.0.0099.ebuild

```diff
@@ -0,0 +1,352 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Please bump with app-editors/vim-core and app-editors/vim
+
+VIM_VERSION="9.0"
+LUA_COMPAT=( lua5-{1,3,4} luajit )
+PYTHON_COMPAT=( python3_{8..11} )
+PYTHON_REQ_USE="threads(+)"
+USE_RUBY="ruby26 ruby27 ruby30 ruby31"
+
+inherit vim-doc flag-o-matic xdg-utils bash-completion-r1 prefix lua-single python-single-r1 ruby-single
```

Comment:
Could we please sort this?

Reply:
I would, but I had originally decided to leave it in order not to mess with phases set by EXPORT_FUNCTIONS.